hooks/persistence-defectdojo/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+	id 'java'
+  id "io.freefair.lombok" version "5.3.0"
+}
+
+group = 'io.securecodebox'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+	mavenCentral()
+  jcenter()
+  maven {
+    url = "https://oss.sonatype.org/content/repositories/snapshots/"
+  }
+}
+
+dependencies {
+  implementation 'io.kubernetes:client-java:10.0.0'
+
+  implementation 'io.securecodebox:defectdojo-client:0.0.7-SNAPSHOT'
```

Comment:
We reference here a released version instead of a snapshot version.

Reply:
Yes 👍
I planned to do this later after this initial merge once we're fairly certain that the API of the client is stable.